src/main/java/scala/compat/java8/runtime/CollectionInternals.java

```diff
@@ -0,0 +1,23 @@
+package scala.compat.java8.runtime;
+
+import scala.collection.immutable.*;  // Don't rely on this!  Refer to everything explicitly!
```

Comment:
Seems to work: https://github.com/szeiger/scala-java8-compat/commit/f190438f698deed09a26013a87acd5617dc54362